docs/cli.md

```diff
@@ -75,9 +28,23 @@ when running in plugin mode, so instead of using `--flag` you must specify a val
 
 # Use
 
-Run `svcat -h` to see the available commands.
+Run `svcat --help` to see the available commands.
 
-Below are some common tasks made easy with svcat. The example output assumes that [User Provided Service Broker](../../charts/ups-broker) is installed on the cluster.
+Below are some common tasks made easy with svcat. The example output assumes that the 
+[User Provided Service Broker](../charts/ups-broker) is installed on the cluster.
+
+* [Find brokers installed on the cluster](#find-brokers-installed-on-the-cluster)
+* [Trigger a sync of a broker's catalog](#trigger-a-sync-of-a-brokers-catalog)
+* [List available service classes](#list-available-service-classes)
+* [View service plans associated with a class](#view-service-plans-associated-with-a-class)
+* [Provision a service](#provision-a-service)
+* [View all instances of a service plan on the cluster](#view-all-instances-of-a-service-plan-on-the-cluster)
+* [List all service instances in a namespace](#list-all-service-instances-in-a-namespace)
+* [Bind an instance](#bind-an-instance)
+* [View the details of a service instance](#view-the-details-of-a-service-instance)
+* [Unbind all applications from an instance](#unbind-all-applications-from-an-instance)
+* [Unbind a single application from an instance](#unbind-a-single-application-from-an-instance)
+* [Delete a service instance](#delete-a-service-instance)
```

Comment:
I can't find where the content for this header is, but is this recursive? Does it delete all the bindings for me or do I have to do that manually before deprovisioning the instance?

Reply:
If you view the md file instead of the diff, you can click on the links. 😀 

https://github.com/carolynvs/service-catalog/blob/svcat-walkthrough2/docs/cli.md#delete-a-service-instance

`svcat deprovision` doesn't delete the bindings for you. There is `svcat unbind INSTANCE_NAME` which does nuke everything for you. It would be nice to add a flag that combined the two, e.g. `svcat deprovision --unbind`. I'm not sure it's a safe default to unbind an instance that's in use, but I could go either way on that.

Reply:
I would add a blurb about that in the thing for deleting a service instance then - "be sure to delete all bindings before deprovisioning the instance" or something.

Reply:
I'm going to add a CLI category to our resources so that you'll be able to do `kubectl delete service-catalog-all` and it will get everything in the ns.

Reply:
That will be so helpful! 💯